appveyor.yml

```diff
@@ -6,10 +6,10 @@ environment:
     - nodejs_version: 6
       platform: x64
       msvs_toolset: 12
-    - nodejs_version: 7
+    - nodejs_version: 8
```

Comment:
Add

```
skip_commits:
  files:
    - docs/*
    - travis.yml
    - '**/*.html'
    - '*.md'
```
very useful only supported on appveyor

Reply:
can you send a PR for this?

Reply:
yes, I can.